Directory.Packages.props

```diff
@@ -4,9 +4,9 @@
     <CentralPackageTransitivePinningEnabled>false</CentralPackageTransitivePinningEnabled>
   </PropertyGroup>
   <ItemGroup>
+    <PackageVersion Include="Acornima" Version="0.9.1" />
```

Comment:
Ahah! nobody will install Jint now because "it's not a major version, blablabla" ;)